docs/framework/install/on-windows-and-server.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Install .NET Framework on Windows
+description: Learn how to install .NET Framework on Windows 11, Windows 10, and Windows Server. This article also includes information about .NET Framework and unsupported versions of Windows, such as Windows 8, Windows Vista, and Windows XP.
+ms.date: 02/12/2025
```

Comment:
Should we add ms.custom:updateeachrelease since this article can get out of date easily?

Reply:
This will just become part of my normal OS updates. I'll have to watch the Server/Client releases for Windows. There shouldn't be anymore .NET Framework releases.